readme.md

```diff
@@ -1,20 +1,39 @@
-Including in Unity project:
+# RenderStream Unity Plugin
 
-*  copy DisguiseUnityRenderStream plugin to the Unity assets folder.
+![alt text](https://download.disguise.one/media/6921/unity.png)
```

Comment:
Is this a permalink or should we rehost the image on github?

Reply:
You're right, should be a permalink, I don't think it is currently.